src/iquest.cpp

```diff
@@ -51,6 +51,15 @@ usage() {
         "in the 'WHERE' condition will be made upper case so one can do case-insensitive",
         "tests (using upper-case literals).",
         " ",
+        "To do a case-insensitive search, pass the 'uppercase' (or 'upper') option and",
+        "make ALL conditional input values uppercase.  For example:",
+        " ",
+        "  iquest uppercase \"SELECT DATA_ID WHERE COLL_NAME LIKE '%/TEST' AND DATA_NAME = 'FOO.BIN'\"",
+        " ",
+        "Case-insensitive search will fail if the WHERE-clause contains columns which have",
+        "an integer-based data type.  That means, columns such as DATA_SIZE will cause the",
+        "query to fail with a CAT_SQL_ERR.",
```

Comment:
It's probably safer/better to say that all columns in the where-clause need to have a string-based type. As it is, some users may think that ONLY integers break the query.

So how about ...
```
Case-insensitive search requires that all columns in the WHERE-clause to have a
string-based data type (e.g. varchar).  That means, columns such as DATA_SIZE
will cause the query to fail with a CAT_SQL_ERR.
```

Reply:
that does seem more inclusive / correct.

Reply:
Done.